lib/whitehall_importer/integrity_checker.rb

```diff
@@ -57,6 +57,7 @@ def image_problems
 
       %w(alt_text caption).each_with_object([]) do |attribute, problems|
         if publishing_api_image[attribute] != proposed_image_payload[attribute]
+          next if attribute == "alt_text" && proposed_image_payload.empty? && publishing_api_image[attribute] == "placeholder"
```

Comment:
I wonder if it might be helpful to extract this out into its own method that describe the logic?  It might not be immediately obvious why we're making this check otherwise.  Something like `#placeholder_image?` maybe?

Reply:
Good idea.  I've moved this into a method, and also added a commit to put the other conditional statement into a named method.